### PR TITLE
alter data field and the form of password encryption

### PR DESCRIPTION
**Pull request by luhaoling**
I have altered the date filed to data layer and changed the form of password encryption.
In this change, I will not insert a new field into the database。

### CODE REVIEW

File: app/user/service/internal/data/user.go [hunk 1]
```diff
@@ -14,6 +16,8 @@ var userTableName = "users"
 type User struct {
 	*biz.User
 	gorm.DeletedAt
+	Created_at time.Time `gorm:"column:created_at"`
+	Updated_at time.Time `gorm:"column:updated_at"`
```
Comment (alilestera) [inline app/user/service/internal/data/user.go]: The field `created_at` and `updated_at` is necessary, right?

**Comment by luhaoling** [comment]
No necessary.The sql file hava realized the date of update.
I will delete the relevant code.

**Comment by luhaoling** [comment]
No necessary.The sql file hava realized the date of update.
I will delete the relevant code.

**Review by alilestera — APPROVED** [review]
Looks GOOD!

**Pull request closed without merging**